django/river/settings/base.py

```diff
@@ -194,7 +195,8 @@
 # https://www.django-rest-framework.org/api-guide/settings/
 
 REST_FRAMEWORK = {
-    "DEFAULT_RENDERER_CLASSES": ["rest_framework.renderers.JSONRenderer"],
+    "DEFAULT_RENDERER_CLASSES": ["djangorestframework_camel_case.render.CamelCaseJSONRenderer"],
```

Comment:
Do you *really* need to change to camel case ?
1. this will be a breaking change for all api consumers.
2. I get that camel case feels more usual in JS, but should we bother to change the case for 1 possible client language ? Snake case is recommended for python and ruby variables.

Reply:
At least if its needed and possible could you make a separate PR ?

Reply:
well we use camelcase also in Golang and Java... And if we chose to share data with JSON which means Javascript object notation I think we better follow the js convention (I tend to prefer gRPC though for communication between services but I know it's a pain in Ruby, don't know with Python). I could apply this renderer class only to Pyrog though ! We tend to have a lot more code on the frontend and it's notoriously harder to maintain so the cleaner syntax is the better.

I can open another PR for this but the rest will depend on it.